src/app/ui/components/containers/headers/header.tsx

```diff
@@ -0,0 +1,92 @@
+import { ReactNode } from 'react';
+
+import { SharedComponentsSelectors } from '@tests/selectors/shared-component.selectors';
+import { Flex, Grid, GridItem, HStack, styled } from 'leather-styles/jsx';
+
+import { ArrowLeftIcon, CloseIcon } from '@app/ui/icons';
+
+import { IconButton } from '../../icon-button/icon-button';
+import { BigTitleHeader } from './components/big-title-header';
+import { HeaderActionButton } from './components/header-action-button';
+
+type HeaderVariants = 'page' | 'home' | 'onboarding' | 'dialog' | 'bigTitle' | 'summary' | 'fund';
+
+export interface HeaderProps {
+  variant: HeaderVariants;
+  iconRight?: ReactNode;
+  isWaitingOnPerformedAction?: boolean;
+  onClose?(): void;
+  onGoBack?(): void;
+  title?: ReactNode;
+  account?: ReactNode;
+  totalBalance?: ReactNode;
+  settingsMenu?: ReactNode;
+  networkBadge?: ReactNode;
+  logo?: ReactNode;
+}
+
+export function Header({
+  variant,
+  iconRight,
+  isWaitingOnPerformedAction,
+  onClose,
+  onGoBack,
+  account,
+  totalBalance,
+  settingsMenu,
+  networkBadge,
+  title,
+  logo,
+}: HeaderProps) {
+  const logoItem = onGoBack || logo || account;
+  iconRight = variant === 'dialog' ? <IconButton icon={<CloseIcon />} onClick={onClose} /> : null;
```

Comment:
@pete-watters I did add the 'summary' variant and the `iconRight` prop to the header to address the `CloseIcon` on the summary pages. But, then I stupidly reset the prop here. Just letting you know bc an easy fix and most the code is in place, but you can obviously change it if you want.

Reply:
I was a bit confused at first with the `getPageVariant()` in the `Container` vs passing the prop when the `Header` is used on its own.

Reply:
Thanks Fara, I'll check and improve this